packages/components/src/accordion/src/Accordion.css

```diff
@@ -41,27 +42,27 @@
     border: 0;
     transition: background-color var(--o-ui-easing-duration-3) var(--o-ui-easing-focus), border-bottom-left-radius 5s linear, border-bottom-right-radius 5s linear;
     width: 100%;
-    color: var(--o-ui-text-alias-primary);
+    color: var(--hop-neutral-text);
     line-height: inherit;
     font-weight: inherit;
     position: relative;
-    padding: 0 var(--o-ui-sp-2);
+    padding: 0 var(--hop-space-inset-sm);
     font-family: inherit;
     font-size: 100%;
 }
 
 /* TRIGGER | BORDERED */
 .o-ui-accordion-bordered .o-ui-accordion-trigger {
-    background-color: var(--o-ui-bg-alias-surface);
-    padding: calc(var(--o-ui-sp-4) - 1px) var(--o-ui-sp-5) var(--o-ui-sp-4) var(--o-ui-sp-6);
-    min-height: var(--o-ui-sz-8);
-    border-top: 1px solid var(--o-ui-b-alias-low-break);
+    background-color: var(--hop-neutral-surface);
+    padding: calc(var(--hop-space-inset-md) - 1px) var(--hop-space-inset-lg) var(--hop-space-inset-md) var(--hop-space-inset-lg);
```

Comment:
What's the point of the -1? Do we still need it?

Reply:
Yep it's linked to the border when selected.